package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "openapi-clientlib",
-    "version": "8.0.3",
+    "version": "8.0.4-ci-extended-asset-types-header-batchrequest",
```

Comment:
just update this to be 8.0.4 and looks good

Reply:
Updated to 8.0.4